src/ibtracert.c

```diff
@@ -67,7 +67,9 @@ static char *node_type_str[] = {
 static int timeout = 0;		/* ms */
 static int force;
 static FILE *f;
+static FILE *lid_fd;
 
+static char *lid_file = NULL;
```

Comment:
parameters could be LIDs or GUIDs.

Reply:
OK - I'll change name and test with lids or guids